dolphinscheduler-standalone-server/src/main/resources/banner.txt

```diff
@@ -0,0 +1,12 @@
+${AnsiColor.CYAN}
+====================================================================================
+ _____        _       _     _          _____      _              _       _
+|  __ \      | |     | |   (_)        / ____|    | |            | |     | |
+| |  | | ___ | |_ __ | |__  _ _ __   | (___   ___| |__   ___  __| |_   _| | ___ _ __
+| |  | |/ _ \| | '_ \| '_ \| | '_ \   \___ \ / __| '_ \ / _ \/ _` | | | | |/ _ \ '__|
+| |__| | (_) | | |_) | | | | | | | |  ____) | (__| | | |  __/ (_| | |_| | |  __/ |
+|_____/ \___/|_| .__/|_| |_|_|_| |_| |_____/ \___|_| |_|\___|\__,_|\__,_|_|\___|_|
+               | |
+               |_|
+=====================================================================================
+Dolphinscheduler 版本: ${application.version}
```

Comment:
I don't suggest using Chinese words in the terminal, which might have unnecessary encoding issue

Reply:
English is better